ts/src/okx.ts

```diff
@@ -4431,12 +4457,13 @@ export default class okx extends Exchange {
         const networksById = this.indexBy (networks, 'id');
         const networkData = this.safeValue (networksById, chain);
         const network = this.safeString (networkData, 'network');
+        const networkCode = this.networkIdToCode (network, code);
```

Comment:
We should check this part as well. Our mappings are used to map OKX's raw network values or CCXT's values?

Reply:
Fixed